tests/python/driver/tvmc/test_frontends.py

```diff
@@ -211,3 +212,43 @@ def test_load_model___wrong_language__to_pytorch(tflite_mobilenet_v1_1_quant):
             model_format="pytorch",
             shape_dict={"input": [1, 3, 224, 224]},
         )
+
+
+@mock.patch("tvm.driver.tvmc.frontends.import_keras")
+def test_import_keras_friendly_message(import_keras_mock, keras_resnet50):
+    import_keras_mock.side_effect = TVMCException("keras is not installed")
```

Comment:
These tests are mostly testing that errors thrown are propagated, it'd be good to test if the errors are actually caused by the import - I think we could use pytest's `monkeypatch` on `__builtin__.__import__` but I'm not feeling very confident in the safety of it. @leandron wdyt?

Reply:
We thought about doing something more elaborate, like simulating the actual import failing, but in the end we settled in just trusting the import error will be there when <input framword of choice> is not there, and work with the behaviour we expect.

Do you have an example on how this monkey would work, so that we can write a more comprehensive test? Otherise I'd keep it as is.

Reply:
it's probably okay to do this right around the call site, but it could get tricky if exception handlers also late-import stuff e.g. pytest's is fairly complex and i don't know what it does entirely.

Reply:
It'd be something like this with `@mock`:
```python
def mock_imports(name, *args):
    if name == 'onnx':
        raise ImportError()
    return orig_import(name, *args)

@mock.patch('__builtin__.__import__', side_effect=mock_imports)
```
Or with `monkeypatch` fixture:
```python
def test_import_keras_friendly_message(import_keras_mock, keras_resnet50, monkeypatch):
  monkeypatch.setattr(__builtin__, '__import__', mock_imports);
```
It's dangerous in either case but given it falls back to the default behaviour for anything else and should be cleaned up after the test it should be stable? We **want** it to catch all imports for any nested `tflite`s or `onnx`s right?